document-generator-prosecution/src/test/java/uk/gov/companieshouse/document/generator/prosecution/handler/ProsecutionHandlerTest.java

```diff
@@ -0,0 +1,168 @@
+package uk.gov.companieshouse.document.generator.prosecution.handler;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import uk.gov.companieshouse.api.model.prosecution.defendant.DefendantApi;
+import uk.gov.companieshouse.api.model.prosecution.offence.OffenceApi;
+import uk.gov.companieshouse.api.model.prosecution.prosecutioncase.ProsecutionCaseApi;
+import uk.gov.companieshouse.api.model.prosecution.prosecutioncase.ProsecutionCaseStatusApi;
+import uk.gov.companieshouse.document.generator.interfaces.model.DocumentInfoResponse;
+import uk.gov.companieshouse.document.generator.prosecution.ProsecutionType;
+import uk.gov.companieshouse.document.generator.prosecution.exception.HandlerException;
+import uk.gov.companieshouse.document.generator.prosecution.exception.ProsecutionServiceException;
+import uk.gov.companieshouse.document.generator.prosecution.mapping.mappers.ApiToDefendantMapper;
+import uk.gov.companieshouse.document.generator.prosecution.mapping.mappers.ApiToOffenceMapper;
+import uk.gov.companieshouse.document.generator.prosecution.mapping.mappers.ApiToProsecutionCaseMapper;
+import uk.gov.companieshouse.document.generator.prosecution.mapping.model.defendant.Defendant;
+import uk.gov.companieshouse.document.generator.prosecution.mapping.model.offence.Offence;
+import uk.gov.companieshouse.document.generator.prosecution.mapping.model.prosecutioncase.ProsecutionCase;
+import uk.gov.companieshouse.document.generator.prosecution.service.ProsecutionService;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@MockitoSettings(strictness = Strictness.LENIENT)
```

Comment:
Why has the strictness been changed to lenient?

Reply:
The strictness setting annotation has been removed (as it is strict by default), and changing that pointed out an incorrect stubbing which has been corrected now